src/Controls/tests/DeviceTests/Elements/Shape/ShapeTests.cs

```diff
@@ -73,8 +74,10 @@ public async Task ShapeUpdatesBrushCorrectly(uint color)
 
 			await InvokeOnMainThreadAsync(async () =>
 			{
-				await CreateHandlerAndAddToWindow<LineHandler>(line, (handler) =>
+				await CreateHandlerAndAddToWindow<LineHandler>(line, async (handler) =>
 				{
+					await Task.Delay(100);
```

Comment:
Can we somehow use AssertionExtentions.Wait?

Reply:
Let+s just copy and have the duplicate code for now?

Reply:
Applied changes to use `AssertionExtensions.Wait`